games-board/lc0/lc0-0.21.1.ebuild

```diff
@@ -0,0 +1,222 @@
+MY_W_SHA="5b535e00f1828556b9199ce7d4dbdbbf91cc8c2f9084d991f9f4ec7d43baa6e5"
+MY_W_URI="https://lczero.org/get_network?sha=${MY_W_SHA}"
+
+# We need openblas-dev libraries if we don't have internally.
+MY_BLAS_BASE_N="libopenblas-base"
+MY_BLAS_DEV_N="libopenblas-dev"
+MY_BLAS_PV="0.3.5"
+MY_BLAS_SRC="http://archive.ubuntu.com/ubuntu/pool/universe/o/openblas"
+MY_BLAS_BASE_URI="${MY_BLAS_SRC}/${MY_BLAS_BASE_N}_${MY_BLAS_PV}+ds-2_amd64.deb"
+MY_BLAS_DEV_URI="${MY_BLAS_SRC}/${MY_BLAS_DEV_N}_${MY_BLAS_PV}+ds-2_amd64.deb"
+
+SRC_URI="https://github.com/LeelaChessZero/lc0/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	${MY_MODULE_URI} -> ${P}-${MY_MODULE_N}.tar.gz
+	${MY_W_URI} -> ${P}-${MY_W_N}_${MY_W_ID}.pb.gz
+	blas-ext? ( ${MY_BLAS_BASE_URI} -> ${P}-${MY_BLAS_BASE_N}-${MY_BLAS_PV}.deb
+		    ${MY_BLAS_DEV_URI} -> ${P}-${MY_BLAS_DEV_N}-${MY_BLAS_PV}.deb
+	)"
+
+DEPEND="
+	dev-libs/protobuf:0=
+	amd-opencl?    ( !switch? ( >=dev-libs/amdgpu-pro-opencl-18.20.684755
+				    x11-drivers/xf86-video-amdgpu ) )
+	mkl?           ( !switch? ( >=sci-libs/mkl-10.0.5.025 ) )
+	nvidia-cudnn?  ( !switch? ( >=dev-libs/cudnn-7.5.0.56 ) )
+	nvidia-opencl? ( !switch? ( >=dev-util/nvidia-cuda-toolkit-10.1.105 ) )"
+
+RDEPEND="${DEPEND}"
+
+BDEPEND="
+	dev-cpp/gtest
+	dev-util/meson
+	virtual/pkgconfig"
+
+BUILD_DIR="${S}/build/release"
+DOCS=( README.md CONTRIBUTING.md COPYING )
+
+src_unpack() {
+	default
+
+	if use blas-ext; then
+	   unpack_deb ${P}-${MY_BLAS_BASE_N}-${MY_BLAS_PV}.deb
+	   unpack_deb ${P}-${MY_BLAS_DEV_N}-${MY_BLAS_PV}.deb
+	fi
+}
+
+src_prepare() {
+	default
+
+	# We need to place submodule for build.
+	cp -r \
+	   "${WORKDIR}"/lczero-common-master/* \
+	   "${S}"/libs/lczero-common/ || die "couldn't import proto module"
+
+	# Fix build issue for 0.21.1
+	sed -i "s*usr/local*usr*g" "$S"/meson_options.txt || die "couldn't set FHS/Gentoo paths"
+
+	if use blas-int; then
+	   # Fix openblas library name for science overlay.
+	   eapply "${FILESDIR}/${P}-science.overlay.patch"
+
+	   # Fix openblas internal library paths.
+	   sed \
+              -e "s*'/usr/lib/'*'/usr/lib/','/usr/lib64/'*g" \
+              -i "$S"/meson_options.txt || die "couldn't set openblas lib paths"
+	fi
+
+	# Prepare to compile with !static! openblas libraries on the fly.
+	# Meson always picks shared if they are in same directory.
+	if use blas-ext; then
+	   rm -rf \
+	         "${WORKDIR}"/usr/lib/x86_64-linux-gnu/libopenblasp-r0.3.5.so \
+	         "${WORKDIR}"/usr/lib/x86_64-linux-gnu/libopenblas.so* || die "couldn't remove shared libraries"
+
+	   # Fix openblas paths for compiling on the fly.
+	   sed \
+	      -e "s*'/usr/lib/'*'${WORKDIR}/usr/lib/x86_64-linux-gnu/'*g" \
+	      -e "s*'/usr/include/openblas/'*'${WORKDIR}/usr/include/x86_64-linux-gnu/'*g" \
+	      -i "$S"/meson_options.txt || die "couldn't set openblas lib paths"
+
+	   # Fix cblas header name.
+	   pushd "${WORKDIR}"/usr/include/x86_64-linux-gnu/ >/dev/null
+	   mv cblas-openblas.h cblas.h || die
```

Comment:
Why not just 
`mv "${WORKDIR}"/usr/include/x86_64-linux-gnu/{cblas-openblas.h,cblas.h} || die`
instead of the push/popd?

Reply:
Ok.